src/internal/stencil-core/index.d.ts

```diff
@@ -50,3 +50,5 @@ export {
 } from '../stencil-public-runtime';
 
 export type { StencilConfig as Config, PrerenderConfig } from '../stencil-public-compiler';
+
+export { CustomElementsExportBehavior } from '../stencil-public-compiler';
```

Comment:
You can't use enums as a value if they're imported as a `type`, so, rather than changing the existing import from a `type` import I added this for the values we need as standard imports.